_components/icon/guidance/when-to-consider-something-else.md

```diff
@@ -1,3 +1,6 @@
-- **Meaning is ambiguous.** Use icons in a common or conventional way only. If the meaning of an icon is ambiguous by itself, it needs accompanying text.
-- **Compensating for page structure.** Icons don’t fix unclear page hierarchy or confusing content organization. Don’t rely on an icon to help draw attention to something important that’s hard to find. Fix the content’s placement first.
-- **You need illustrative artwork.** Icons have a specific, functional meaning. Avoid using icons outside their intended usage for illustrative purposes (like using the “visibility” eye icon to illustrate an actual eye). If you want to use a graphic to enhance your content or explain a more in-depth concept, consider using an illustration instead of an icon.
\ No newline at end of file
+{% capture visibility_icon %}<svg class="usa-icon bottom-neg-2px" aria-hidden="true" focusable="false" role="img"><use xlink:href="{{ site.baseurl }}/assets/img/sprite.svg#visibility"></use></svg>{% endcapture %}
+
+{:.usa-content-list }
+- **Meaning is ambiguous.** Use icons only in a common or conventional way. Icon utility hinges on people quickly recognizing what each icon means. If you suspect that an icon’s intent isn’t perfectly clear, consider removing it or adding accompanying text.
+- **Compensating for page structure.** Don’t rely on an icon to help draw attention to something important that’s otherwise hard to find. Icons don’t fix unclear page hierarchy or confusing content organization.
+- **You need illustrative artwork.** Icons have a specific, functional meaning. Avoid using icons for illustrative purposes (like using the “visibility” eye icon ({{ visibility_icon }}) to illustrate an actual eye). If you want to enhance the appearance of your content or visually explain a concept, use an illustration instead of an icon.
```

Comment:
@jaclinec @sarah-sch @finekatie 

I wasn't sure the best method for adding the icon example here. Right now it is in a parenthesis inside a parenthesis. Could we replace the outer parenthesis with a comma? Let me know what is best here!

![image](https://github.com/uswds/uswds-site/assets/93996430/6cbac839-c675-40fe-abe9-2414ec0ee354)

```suggestion
- **You need illustrative artwork.** Icons have a specific, functional meaning. Avoid using icons for illustrative purposes, like using the “visibility” eye icon ({{ visibility_icon }}) to illustrate an actual eye. If you want to enhance the appearance of your content or visually explain a concept, use an illustration instead of an icon.
```

Reply:
Do we need the icon inside parentheses or could we just have it kind of floating out there? I don't want people to think the parentheses are part of the icon itself.

Reply:
@amyleadem @sarah-sch Sarah's suggestion works for me!

Reply:
I updated this copy according to @sarah-sch's suggestion in [a7d9334](https://github.com/uswds/uswds-site/pull/2643/commits/a7d93341e87c97147a1f7f8219a976a64d9b35b9). Please let me know if you need any changes.